test/jdk/java/nio/file/Path/PathOps.java

```diff
@@ -1437,6 +1437,34 @@ static void doWindowsTests() {
         int h2 = test("c:\\FOO").path().hashCode();
         if (h1 != h2)
             throw new RuntimeException("PathOps failed");
+
+        // long path prefixes
+        test("\\\\?\\C:\\mnt\\file.dat")  // absolute
+            .string("C:\\mnt\\file.dat");
+        test("\\\\?\\\\\\server\\share\\dir\\file.dat")  // UNC
+            .invalid();
+        test("\\\\?\\file.dat")           // relative
+            .invalid();
+        test("\\\\?\\\\file.dat")         // directory-relative
+            .invalid();
+        test("\\\\?\\C:file.dat")         // drive-relative
+            .invalid();
+        test("\\\\?\\")                   // empty
+            .invalid();
+
+        // long UNC path prefixes
+        test("\\\\?\\UNC\\server\\share\\dir\\file.dat")      // UNC
+            .string("\\\\server\\share\\dir\\file.dat");
+        test("\\\\?\\UNC\\server\\share\\C:\\file.dat")       // absolute
+            .invalid();
+        test("\\\\?\\UNC\\file.dat")                          // relative
+            .invalid();
+        test("\\\\?\\UNC\\server\\share\\C:file.dat")         // drive-relative
+            .invalid();
+        test("\\\\?\\UNC")                                    // empty
+            .invalid();
+        test("\\\\?\\UNC\\")                                    // empty
```

Comment:
The comment on this line isn't aligned with the other but the updated list of test cases looks good/complete.

Reply:
Aligned in f9de5966fe505249fb373bcd8551183977e5e8e3.